mycroft/skills/diagnostics/vocab/en-us/IpKeyword.voc

```diff
@@ -0,0 +1,10 @@
+Ip
+What is my ip?
+What is my eye pea?
+What is my eye pee?
+What is my I pea?
+What is my I pee?
+I pea
+I pee
+eye pea
+eye pee
```

Comment:
You shouldn't need any of these extra possible pronunciations, as the stt engine should ideally give one canonical form of a given utterance.